set-environment

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+#enabled="$(snapcraft get enabled)"
+#if [ -z "${enabled}" || "${enabled}" != "true" ]
+#    echo "Plume not yet enabled"
+#    exit -1
+#fi
```

Comment:
why is this commented out?

Reply:
Because I forgot to remove it, apparently!

Reply:
It would be good to fix this up so that the server doesn't try to start until setup the been completed. This *was* a manual attempt at doing that, but didn't quite work.

Reply:
Done! Plume now doesn't try to run until it's been enabled, and doesn't let you enable it until you've set the minimum configuration items.